app/controllers/report_controller.rb

```diff
@@ -0,0 +1,44 @@
+class ReportController < ApplicationController
+  def select_soc_codes
+    report = Report.where(guid: params[:guid]).first
+    @keywords = report.keywords
+  end
+
+  def save_soc_codes
```

Comment:
"lol yes" actually :) writing some tests at the moment which has driven doing exactly that :)